.travis.yml

```diff
@@ -30,11 +34,11 @@ before_install:
 install:
   - gcloud -q components install kubectl
   - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.16.0
-  - go get -t ./...
+  - if [ "$GO111MODULE" = "auto" ]; then go get -t -v ./... ; fi
 
 script:
   - go build ./...
-  - golangci-lint run
+  - golangci-lint run --deadline=5m
```

Comment:
Is the timeout necessary both here and in .golangci.yml?

Reply:
Testing showed that the command didn't pick up the one in golangci.yml

Reply:
Strange. What effect does the setting in golangci.yml have? If none, we should probably remove it.